README.md

```diff
@@ -94,6 +94,12 @@ making the combined instantiation/stub declaration very terse:
 double(:foo => 'bar')
 ```
 
+So, the above examples can be written simply as:
+
+```ruby
+allow(book).to receive(:title).and_return(book)
+```
+
```

Comment:
This doesn't make it clearer at all to me, the examples above are already rewriting this form, in fact the first examples cover this exact syntax (without looping back to `book`). Can you explain what you're clarifying here?